recipe/meta.yaml

```diff
@@ -24,7 +24,7 @@ requirements:
   run:
     - python >=3.6
     - conda-build >=3.24,<3.26
```

Comment:
```suggestion
    - conda-build >=3.24
```

Reply:
Why unpin `conda-build`? This change is one of the causes for https://github.com/mamba-org/boa/issues/388.

Reply:
Because that was a long-running effort to unblock many other things that required newer conda-build. The previous cap was there due to explicit incompatibilities, which were removed, hence the cap could be removed.

Now a _new_ incompatibility got introduced with 3.28. It's a separate topic. The only take-away perhaps is to be more defensive on the pins here, and use `<={last_working_conda_build}`. In an ideal world that shouldn't be necessary (conda-build should deprecate behaviour changes, rather than just break things), but well...

Reply:
I see, thanks for clarifying.

Lately my practice has been to pin on major versions (i.e. `conda-build >=3.24,<4`) as those are likely to come with breaking changes. However, as we see in this case, there's still the possibility of breaking changes coming in minor version updates, which is unfortunate...

Reply:
Not everyone does versioning according to the server spec...